misc/systemd/cf-consumer.service.in

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=CFEngine Enterprise Report Consumer
+PartOf=cfengine3.service
+After=syslog.target
+After=cf-postgres.service
+After=cf-redis-server.service
+ConditionPathExists=@bindir@/cf-consumer
+BindsTo=cf-redis-server.service
+BindsTo=cf-postgres.service
+
+[Service]
+Type=simple
+ExecStart=@bindir@/cf-consumer --no-fork --verbose
```

Comment:
I think I would stick with `@workdir@/bin` for now. `@sbindir@` probably will work, but if we start using it we should use it everywhere. I don't have a strong preference though.